index.html

```diff
@@ -0,0 +1,163 @@
+            <div class="header_squiggle"></div>
+        </div>
+    </div>
+
+</header>
+
+
+<main>
+    <section class="outer-container">
+        <section class="inner-container inner-container-main">
+            <div class="main--container--top">
+                <h2>Rekupe is a digital agency in Los Angeles. </h2>
+                <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
+                <h2> We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+                    Have a question? Click here to contact us!</h2>
+                <p>Have a question? <a class="main--link" href="#">Click here</a> to contact us!</p>
+            </div>
+        </section>
+
+        <section class="inner-container">
+            <section class="article-container">
+                <div class="main--container--article">
+                    <article>
+                        <h3> Rolling stone</h3>
+                        <img class="main--img--squiggle" src="assets/squiggle_line.svg" alt="Squiggle line"/>
+                        <p>You know Rolling Stone. It's the definitive voice in music, politics and culture it's a
+                            global icon.</p>
+                        <a class="main--link" href="#">VIEW PROJECT</a>
+                    </article>
+                    <span>
+        <img src="assets/work-image-01.jpeg" alt="RollingSTone on a laptop">
+    </span>
+                </div>
+
+                <div class="main--container--article">
+    <span>
+        <img src="assets/work-image-02.jpeg" alt="Photo of variety magazine">
+    </span>
+                    <article>
+                        <h3> Variety </h3>
+                        <img class="main--img--squiggle" src="assets/squiggle_line.svg" alt="Squiggle line"/>
+                        <p>Variety is an industry leader for breaking entertainment news, review and box office results
+                            with archive of content dating back to 1905</p>
+                        <a class="main--link" href="#">VIEW PROJECT</a>
+                    </article>
+                </div>
+
+                <div class="main--container--article">
+                    <article>
+                        <h3>Worth</h3>
+                        <img class="main--img--squiggle" src="assets/squiggle_line.svg" alt="Squiggle line"/>
+                        <p>A global media platform connection a community that embraces worth beyond wealth</p>
+                        <a class="main--link" href="#">VIEW PROJECT</a>
+                    </article>
+                    <span>
+        <img src="assets/work-image-03.jpeg" alt="Photo of Variety500">
+    </span>
+                </div>
+
+            </section>
+
+            <div class="main--container--article--border--flex">
+                <article class="main--container--article--border"><img src="assets/icon_web-design.svg"
```

Comment:
Als ik het scherm heel smal maak, dan vind ik dat er nog wel wat te veel witruimte is
![Scherm­afbeelding 2023-01-20 om 21 14 35](https://user-images.githubusercontent.com/23332341/213797380-7a6edaee-5b53-4854-9763-5628a3c39697.jpg)